frontend/src/components/pages/student/AssessmentSummaryPage.tsx

```diff
@@ -33,6 +34,8 @@ const AssessmentSummaryPage = (): React.ReactElement => {
     history.push(WRITE_ASSESSMENT_PAGE);
   };
 
+  usePageTitle(test?.name ?? "Assessment Summary");
```

Comment:
this is defaulting to `"Assessment Summary"` on initial load and only if i press start + go back then it shows the test name. it's a small detail but just wondering why that is.

would it work if we pass in `usePageTitle(test?.name, "Assessment Summary");` separately and wait for `test?.name` value to change - and if falsy then we set to second param